R/available-packages.R

```diff
@@ -89,13 +89,10 @@ renv_available_packages_query <- function(type, repos, quiet = FALSE) {
   if (empty(errors))
     return(dbs)
 
-  if (!is_testing())
```

Comment:
I wonder if `renv_pretty_print` needs a new name, and if it should just write output by default. We almost always use it for "warning"-style output.